src/main/java/com/keyboardbuilder/enterprise/dto/Build.java

```diff
@@ -2,20 +2,23 @@
 
 import com.google.gson.annotations.SerializedName;
 import lombok.Data;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public @Data
```

Comment:
Fixed build error since public keyword was missing and this class is imported elsewhere.

Reply:
Good.